aggregate_root/lib/aggregate_root/snapshot_repository.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+require 'base64'
+require 'ruby_event_store/event'
+
+module AggregateRoot
+  class SnapshotRepository
+    def initialize(event_store, interval = 1)
```

Comment:
I'd set default interval to 100. That's arbitrary value but also a default fot batched reads. Then I'd explicitly pass 1 to SnapshotRepository in specs.